glean-core/android/src/test/java/mozilla/telemetry/glean/private/LabeledMetricTypeTest.kt

```diff
@@ -397,15 +397,15 @@ class LabeledMetricTypeTest {
 
     @Test(expected = IllegalStateException::class)
     fun `Test that labeled events are an exception`() {
-        val eventMetric = EventMetricType<NoExtraKeys>(
+        val eventMetric = EventMetricType<NoReasonCodes>(
             disabled = false,
             category = "telemetry",
             lifetime = Lifetime.Application,
             name = "labeled_event_metric",
             sendInPings = listOf("metrics")
         )
 
-        val labeledEventMetric = LabeledMetricType<EventMetricType<NoExtraKeys>>(
+        val labeledEventMetric = LabeledMetricType<EventMetricType<NoReasonCodes>>(
```

Comment:
I don't think you meant to change this here, I think `NoExtraKeys` is correct for an `EventMetricType`.